code/style.css

```diff
@@ -1,7 +1,292 @@
+}
+
+.form__box {
+  display: grid;
+  grid-template-columns: 1fr;
+  background-color: #fff;
+  box-shadow: 0 5px 10px 0 rgba(0,0,0,0.19);
+}
+
+.form__box-left {
+  background: url("./assets/baking-img2.jpeg");
+  background-position: center;
+  background-size: cover;
+  height: 60vh;
+}
+
+.form__box-right {
+  padding: 20px;
+}
+
+.form__title {
+  display: flex;
+  justify-content: center;
+  font-size: 26px;
+  font-weight: bold;
+  color: #f865b1;
+  margin: 10px 0 25px 0;
+}
+
+.form__input,
+.form__textarea {
+	width: 100%;
+	border: none;
+	background-color: rgba(230, 230, 230, 0.6);
+	padding: 12px;
+	font-size: 16px;
+	margin: 7px 0 22px 0;
+}
+
+.form__input:focus {
+	background-color: rgba(0, 0, 0, 0.1);
+}
+
+.form__input:valid { 
+  border: 1px solid #5b9ea5;
+}
+
+.form__input:invalid {
+  border-bottom: 1px solid #F865B0;
+}
+
+.form__selectors {
+  display: flex;
+  flex-direction: column;
+}
+
+.form__selectors .form__radio-button {
+  margin: 8px;
+  cursor: pointer;
+}
+
+.form__checkbox {
+  margin-top: 14px;
+  display: flex;
+  align-items: center;
+}
+
+input[type="checkbox"] {
+  appearance: none;
+  height: 18px;
+  width: 18px;
+  margin-right: 15px;
+  background-color: rgba(230, 230, 230, 0.6);
+  display: flex;
+  justify-content: center;
+}
+
+input[type="checkbox"]::after {
+  content: "\2573";
+  display: none;
+}
+
+input[type="checkbox"]:checked::after {
+  display: block;
+} 
+
+.form__button {
+  width: 100%;
+  cursor: pointer;
+  margin-top: 20px;
+  padding: 10px 16px;
+  border: none;
+  border-radius: 8px;
+  color: #f9f9f9;
+  text-transform: uppercase;
+  font-weight: bold;
+  letter-spacing: 1px;
+  background-color: #5b9ea5;
+  transition: background-color 0.5s ease-in-out;
+}
+
+.form__button:hover {
+  background-color: #F865B0;
+}
+
+/*** ACCORDION ***/
+.accordion {
+  width: 100%;
+  height: 700px;
+  background: #dac3d0; 
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.accordion__header {
+  text-align: center;
+  font-size: 26px;
+  color: #F865B0;
+}
+
+.accordion__item {
+  margin: 10px;
+  background-color: #fff;
+  border-radius: 7px;
+  box-shadow: 0 5px 10px 0 rgba(0,0,0,0.19);
+}
+
+.accordion__title {
+  padding: 15px 15px 15px 10px;
+  line-height: 25px;
+  font-weight: bold;
+  position: relative;
+  cursor: pointer;
+}
+
+.accordion__title::after {
+  content: "\002B";
+  position: absolute;
+  right: 10px;
+  transition: 0.2s ease-in-out;
+}
+
+.accordion__title.active::after {
+ transform: rotate(45deg);
+}
+
+.accordion__content {
+  overflow: hidden;
+  height: 0;
+}
+
+.accordion__content-answer {
+  border-top: 2px solid #dac3d0;
+  padding: 15px 15px 15px 12px;
+  line-height: 25px;
+}
+
+.active + .accordion__content {
+  height: auto;
+}
+
+/*** FOOTER ***/ 
+.footer {
+  background-color: #5b9ea5;
+  height: 100px;
+}
+
+.footer__icons {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer__icons .fab, 
+.fas {
```

Comment:
I'm not sure if this is what you wanted to achieve. If you wanted to style `.fab` and `.fas` elements that are nested inside `.footer__icons`, then we should define it like this :

```
.footer__icons .fab, 
.footer__icons .fas {
  // styles here
}
```